qt/exclude_list_dialog.py

```diff
@@ -118,8 +118,6 @@ def onTestStringButtonClicked(self):
             return
         # if at least one row matched, we know whether table is highlighted or not
         self._row_matched = self.model.test_string(input_text)
-        # FIXME There is a bug on Windows (7) where the table rows don't get
-        # repainted until the table receives a mouse click event.
```

Comment:
I'm still seeing this behavior in windows 10 clicking Test String does not highlight the table until clicking on the table.

Reply:
Which Qt5/pyqt5 versions are you using?

Reply:
From my testing on Linux, when I run pyqt 5.15.1 there is no bug, but when I use 5.14 the bug occurs.
Tested in a venv with `pip install pyqt5==5.14`.

Reply:
This is with pyqt5 5.15.2 on windows 10.

Reply:
Thanks, should work now.